### ISSUE TITLE
Specfile bug: Incorrect instance types.

### ISSUE THREAD

**Issue by agamzn** [aws-cloudformation/cfn-python-lint#699]
The current specfiles are reporting invalid instance types. 3xlarge, 6xlarge, 9xlarge, etc, are not valid EC2 Instance types. 

```
 ~/development/cfn-python-lint     master  egrep -i ‘\.(3|6|9|18)xlarge’ -R * | awk ‘{print $2}’ | sort -rn | uniq -c
 12 “z1d.6xlarge”,
 12 “z1d.3xlarge”,
 10 “c5n.9xlarge”,
 10 “c5n.18xlarge”,
 32 “c5d.9xlarge”,
 32 “c5d.18xlarge”,
 36 “c5.9xlarge”,
 36 “c5.18xlarge”,

```

This is pervasive across both pricing and regional specfiles. 

```
~/development/cfn-python-lint     master  egrep -i ‘\.(3|6|9|18)xlarge’ -R * -l
src/cfnlint/data/CloudSpecs/ap-northeast-1.json
src/cfnlint/data/CloudSpecs/ap-northeast-2.json
src/cfnlint/data/CloudSpecs/ap-south-1.json
src/cfnlint/data/CloudSpecs/ap-southeast-1.json
src/cfnlint/data/CloudSpecs/ap-southeast-2.json
src/cfnlint/data/CloudSpecs/ca-central-1.json
src/cfnlint/data/CloudSpecs/eu-central-1.json
src/cfnlint/data/CloudSpecs/eu-north-1.json
src/cfnlint/data/CloudSpecs/eu-west-1.json
src/cfnlint/data/CloudSpecs/eu-west-2.json
src/cfnlint/data/CloudSpecs/eu-west-3.json
src/cfnlint/data/CloudSpecs/sa-east-1.json
src/cfnlint/data/CloudSpecs/us-east-1.json
src/cfnlint/data/CloudSpecs/us-east-2.json
src/cfnlint/data/CloudSpecs/us-gov-east-1.json
src/cfnlint/data/CloudSpecs/us-gov-west-1.json
src/cfnlint/data/CloudSpecs/us-west-1.json
src/cfnlint/data/CloudSpecs/us-west-2.json
src/cfnlint/data/ExtendedSpecs/ap-northeast-1/05_pricing_property_values.json
src/cfnlint/data/ExtendedSpecs/ap-northeast-2/05_pricing_property_values.json
src/cfnlint/data/ExtendedSpecs/ap-south-1/05_pricing_property_values.json
src/cfnlint/data/ExtendedSpecs/ap-southeast-1/05_pricing_property_values.json
src/cfnlint/data/ExtendedSpecs/ap-southeast-2/05_pricing_property_values.json
src/cfnlint/data/ExtendedSpecs/ca-central-1/05_pricing_property_values.json
src/cfnlint/data/ExtendedSpecs/eu-central-1/05_pricing_property_values.json
src/cfnlint/data/ExtendedSpecs/eu-north-1/05_pricing_property_values.json
src/cfnlint/data/ExtendedSpecs/eu-west-1/05_pricing_property_values.json
src/cfnlint/data/ExtendedSpecs/eu-west-2/05_pricing_property_values.json
src/cfnlint/data/ExtendedSpecs/eu-west-3/05_pricing_property_values.json
src/cfnlint/data/ExtendedSpecs/sa-east-1/05_pricing_property_values.json
src/cfnlint/data/ExtendedSpecs/us-east-1/05_pricing_property_values.json
src/cfnlint/data/ExtendedSpecs/us-east-2/05_pricing_property_values.json
src/cfnlint/data/ExtendedSpecs/us-gov-east-1/05_pricing_property_values.json
src/cfnlint/data/ExtendedSpecs/us-gov-west-1/05_pricing_property_values.json
src/cfnlint/data/ExtendedSpecs/us-west-1/05_pricing_property_values.json
src/cfnlint/data/ExtendedSpecs/us-west-2/05_pricing_property_values.json

```

**Comment by cmmeyer** [comment]
I'm confused what the problem is? These seem to be legitimate instance types and are all listed on the EC2 product pages:

Here's the c5 types:
https://aws.amazon.com/ec2/instance-types/c5/

```
c5.9xlarge | 36 | 72 | EBS-Only | 10 | 7,000
c5.18xlarge | 72 | 144 | EBS-Only | 25 | 14,000
c5d.9xlarge | 36 | 72 | 1 x 900 NVMe SSD | 10 | 7,000
c5d.18xlarge | 72 | 144 | 2 x 900 NVMe SSD | 25 | 14,000
c5n.9xlarge | 36 | 96 | EBS-Only | 50 | 7,000
c5n.18xlarge | 72 | 192 | EBS-Only | 100 | 14,000
```

It takes a little digging, but the z1d types are here:
https://aws.amazon.com/ec2/instance-types/

```
z1d.3xlarge | 12 | 96 | Up to 10 Gigabit | 1 x 450 NVMe SSD
z1d.6xlarge | 24 | 192 | 10 Gigabit | 1 x 900 NVMe SSD
```

Perhaps they are only available in some regions? The EC2 page doesn't specify.
